README.md

```diff
@@ -81,29 +81,29 @@ Assume ceph is installed on all nodes, this part is demonstrate the workflow of
   KEY | VALUE | DESCRIPTION
 ------|-------|------------
   clean build | true / false | Set true, clean current deployed ceph and redeploy a new cluster; Set false, try obtain current cluster layout, and add new osd to the existing cluster.
-  head | ${hostname} | Cetune controller node hostname
-  user | root | Only support root currently
-  enable_rgw | true / false | Set true, cetune will also deploy radosgw; Set false, only deploy osd and rbd nodes
-  list_server | ${hostname1},${hostname2},... | List osd nodes here, split by ','
-  list_client | ${hostname1},${hostname2},... | List client(rbd/cosbench worker) nodes here, split by ','
-  list_mon | ${hostname1},${hostname2},... | List mon nodes here, split by ','
-  ${server_name} | ${osd_device1}:${journal_device1},${osd_device2}:${journal_device2},... | After adding nodes at 'list_server', cetune will add new lines whose key is the server's name;Add osd:journal pair to corresponding node, split by ','
+  head | ${hostname} | Cetune controller node hostname.
+  user | root | Only support root currently.
+  enable_rgw | true / false | Set true, cetune will also deploy radosgw; Set false, only deploy osd and rbd nodes.
+  list_server | ${hostname1},${hostname2},... | List osd nodes here, split by ','.
+  list_client | ${hostname1},${hostname2},... | List client(rbd/cosbench worker) nodes here, split by ','.
+  list_mon | ${hostname1},${hostname2},... | List mon nodes here, split by ','.
+  ${server_name} | ${osd_device1}:${journal_device1},${osd_device2}:${journal_device2},... | After adding nodes at 'list_server', cetune will add new lines whose key is the server's name;Add osd:journal pair to corresponding node, split by ','.
```

Comment:
we don't need '.' here, it shows inside the third colume of the table, and the sentence is not complete.